### DISCUSSION TITLE
Is there a way to put date/time in the filename?

### DISCUSSION THREAD

**Discussion by POD-io** [mifi/lossless-cut#1520] [Q&A] [answered]
I like a very simple filename on my output, but right now it's so simple that I'm overwriting my files. Would like to add a very short extension to the filename, something like the current time. I didn't see these variables available in the program, are they perhaps hidden? Otherwise, any recommendation on what I can add to keep my filename short but descriptive?

**Comment by mifi** [comment]
> I didn't see these variables available in the program, are they perhaps hidden?

see Output file names in the "export options" dialog, is that what you're looking for?

<img width="722" alt="Screenshot 2023-03-28 at 14 15 29" src="https://user-images.githubusercontent.com/402547/228135034-9fa1ab65-1397-4fc1-adcb-7271a79b29c0.png">

**Reply by POD-io** [reply]
Thanks but I mean specifically date or time options -- I did see those but I was unsure if those were all inclusive. I tried to search documentation but had no luck, hoping someone may be aware of 'hidden' variables or additional ones that weren't listed there.

**Reply by mifi** [reply]
Ah ok. So you want like wall clock time? Time that the export started? So all segments will have the same time value?

**Reply by POD-io** [reply]
That would be pretty good. I've typically seen other programs have options to use values ranging from year/month down to minute/second, or even ms.

For example, if I could specify something like ${HHMMSS} it would output 193240 -- representing 07:32PM and 40 seconds in this case. That allows me to create a file that will have a very unique name, but not too long, and if I make a small adjustment I don't have to worry about overwriting one.

**Comment by mifi** [comment, marked as answer]
I created a feature request from this discussion
